cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,103 @@
 /// <reference types='cypress' />
+const { generateWorker } = require('../support/generateWorker');
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let worker;
 
+  beforeEach(() => {
+    cy.task('generateWorker').then((generateWorker) => {
+      worker = generateWorker;
+      cy.visit('/');
+    });
+  });
+
+  it('should check pagination', () => {
+    cy.get('.-pageInfo').should('exist');
+    cy.get('[aria-label="jump to page"]').type('1');
+  });
+
+  it('should check rows count selection', () => {
+    cy.get('[aria-label="rows per page"]').should('exist');
+    cy.get('[aria-label="rows per page"]').select('20 rows');
+  });
+
+  it('should add a new worker', () => {
+    cy.get('#addNewRecordButton').click();
+    cy.get('#registration-form-modal').should('contain', 'Registration Form');
+    cy.findByPlaceholder('First Name').type(worker.firstName);
+    cy.findByPlaceholder('Last Name').type(worker.lastName);
+    cy.findByPlaceholder('name@example.com').type(worker.email);
+    cy.findByPlaceholder('Age').type(worker.age);
+    cy.findByPlaceholder('Salary').type(worker.salary);
+    cy.findByPlaceholder('Department').type(worker.department);
+    cy.get('#submit').click();
+    cy.get('.rt-td').should('contain', worker.firstName);
+    cy.get('.rt-td').should('contain', worker.lastName);
+  });
+
+  it('should be able to belete a worker', () => {
+    cy.createWorker(generateWorker());
+    cy.get('#delete-record-4').click();
+    cy.contains('.rt-td', generateWorker).should('not.exist');
```

Comment:
this is an incorrect assertion. You need to get table and assert that table does not contain deleted worker
![image](https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/85580286/33b2ed7e-4a7d-4d62-9169-b2f524d8ac2f)

Reply:
Done!